src/legacy/ui/public/agg_types/buckets/terms.js

```diff
@@ -24,7 +24,9 @@ import { AggConfig } from '../../vis/agg_config';
 import { Schemas } from '../../vis/editors/default/schemas';
 import { createFilterTerms } from './create_filter/terms';
 import orderAggTemplate from '../controls/order_agg.html';
-import orderAndSizeTemplate from '../controls/order_and_size.html';
+import { OrderParamEditor } from '../controls/order';
+import { SizeParamEditor } from '../controls/size';
+import { wrapWithInlineComp } from './_inline_comp_wrapper';
```

Comment:
Let's also remove unused dependency `AggConfig` in line 23.

Reply:
Why did you decide that it is an unused dependency?
It is used in line 125 in this file:
```
{
      name: 'orderAgg',
      type: AggConfig,
      default: null,
```
More than that, the CI would be failed, if it was unused.

Reply:
Oh, sorry, it was something wrong with my IDE.